src/operator/time_series/radix_tree/updater.rs

```diff
@@ -0,0 +1,406 @@
+use super::{ChildPtr, Prefix, RadixTreeCursor, TreeNode, RADIX_BITS};
```

Comment:
perhaps I missed it, but are there tests that compare tree-based aggregation with brute-force aggregation for correctness?

Reply:
Yes, `validate` does this.